client/components/data/query-post-likes/index.jsx

```diff
@@ -32,25 +30,13 @@ class QueryPostLikes extends Component {
 		}
 	}
 
-	request( props ) {
-		const { isRequesting, postId, siteId } = props;
-		if ( isRequesting ) {
-			return;
-		}
-
-		props.requestPostLikes( siteId, postId );
+	request( { requestPostLikes: rpl, siteId, postId } ) {
+		rpl( siteId, postId );
```

Comment:
❓ `rpl` --> `requestPostLikes` ❓

Reply:
yeah eslint was complaining about shadowing the imported requestPostLikes